src/schema/v2/CollectorProfile/collectorProfile.ts

```diff
@@ -65,6 +69,43 @@ export const CollectorProfileFields: GraphQLFieldConfigMap<
       return meUserInterestsLoader?.().then(({ body }) => body)
     },
   },
+  interestsConnection: {
+    type: UserInterestConnection,
+    args: pageable({}),
+    resolve: async (
+      { id, partnerId },
+      args,
+      { partnerCollectorProfileUserInterestsLoader }
+    ) => {
+      if (!partnerCollectorProfileUserInterestsLoader || !partnerId) {
+        return null
+      }
+
+      const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
```

Comment:
**question (non-blocking):** Will we need to sort the results on the client side? If so, we may want to include that in the loader.